### DISCUSSION TITLE
Remove followers trophies from the README.md ?

### DISCUSSION THREAD

**Discussion by parmeshwar1820** [anuraghazra/github-readme-stats#2209] [Q&A] [answered]
Remove followers trophies from the README.md ?

**Comment by DURGESH716** [comment, marked as answer]
https://github-profile-trophy.vercel.app/?username=parmeshwar1820&rank=-C,-B
you can simply exclude the particular ranks

**Reply by rickstaa** [reply]
I didn't know we had trophies. 😅 I think this question was meant to be asked at https://github.com/ryo-ma/github-profile-trophy.
